### PR TITLE
Update release notes draft to version v1.22.0-alpha.1

### PR DESCRIPTION
**Pull request by pmmalinov01**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This PR updates the Release Notes Draft to k/k v1.22.0-alpha.1

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
This is an automated PR generated from `krel The Kubernetes Release Toolbox`

### CODE REVIEW

**Comment by pmmalinov01** [comment]
Due to not unidentified problem yet, I can not do all pull request in one release note session.
I am hitting:
`FATA creating Draft PR: while running release notes fix flow: while editing release note: while launching editor: there was a problem with the editor “vi” `
I will try to do them in batches and try to resolve what is happening.

**Comment by puerco** [comment]
Are you using `vi` as your editor? Or did it try to launch it as a default? Did you use it to edit all the map files in the PR or another one?

**Comment by pmmalinov01** [comment]
@puerco I do use `vi` as editor. I used it to edit all maps and it crashes at a random point. The bigger issue is that it does not generate the notes draft markdown and JSON after such crash, and sequential runs of krel release-notes do not pick the session that I stopped at, or crashed. I already rebuild the tool a few times but no change.

**Review by Damans227 — APPROVED** [review]

File: releases/release-1.22/release-notes/maps/pr-100139-map.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+pr: 100139
+releasenote:
+  text: The feature gate `SSA` graduated to GA in v1.22 and therefore will be unconditionally
```
Comment (wilsonehusin) [inline releases/release-1.22/release-notes/maps/pr-100139-map.yaml:3]:
```suggestion
  text: The feature gate `SSA`has graduated to GA in v1.22 and therefore is unconditionally
```

File: releases/release-1.22/release-notes/maps/pr-100267-map.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+pr: 100267
+releasenote:
+  text: It adds two flags, `--max-pods` and `--extended-resources`.
```
Comment (wilsonehusin) [inline releases/release-1.22/release-notes/maps/pr-100267-map.yaml:3]:
```suggestion
  text: Adds two kubemark flags, `--max-pods` and `--extended-resources`.
```

File: releases/release-1.22/release-notes/maps/pr-101171-map.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+pr: 101171
+releasenote:
+  text: ' The new flag `--chunk-size=SIZE` has been promoted to beta, and enabled
+    by default. This flag may be used to alter the number of items or disable this
+    feature when `0` is passed.'
```
Comment (wilsonehusin) [inline releases/release-1.22/release-notes/maps/pr-101171-map.yaml:5]: This looks like SIG CLI, which has different promotion mechanism. I would ping `#sig-cli` to clarify "beta, and enabled by default" (the enabled-by-default policy is usually server-side feature gate which can be enforced).

```suggestion
  text: ' The new flag `--chunk-size=SIZE` has been added to `kubectl describe`.
    This flag may be used to alter the number of items or disable this feature when
    `0` is passed.'
```

File: releases/release-1.22/release-notes/maps/pr-100148-map.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+pr: 100148
+releasenote:
+  text: The new flag `--chunk-size=SIZE` has been promoted to beta, and enabled by
+    default. This flag  may be used to alter the number of items or disable this feature
+    when `0` is passed.
```
Comment (wilsonehusin) [inline releases/release-1.22/release-notes/maps/pr-100148-map.yaml:5]: Also check with `#sig-cli` on wording here.

```suggestion
  text: The new flag `--chunk-size=SIZE` for `kubectl drain` has been promoted to beta,
    and enabled by default. This flag may be used to alter the number of items or disable
    this feature when `0` is passed.
```

**Review by wilsonehusin — COMMENTED** [review]
small notes -- looks good otherwise!

File: releases/release-1.22/release-notes/maps/pr-100694-map.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+pr: 100694
+releasenote:
+  text: 'Fix: not deleting existing publich IP when service is deleted in Azure'
```
Comment (erismaster) [inline releases/release-1.22/release-notes/maps/pr-100694-map.yaml:3]: @pmmalinov01 Should there be single quotes around the entire text body?

File: releases/release-1.22/release-notes/maps/pr-101110-map.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+pr: 101110
+releasenote:
+  text: 'kubeadm: enable `--experimental-patches` flag for `kubeadm join phase control-plane-join
```
Comment (erismaster) [inline releases/release-1.22/release-notes/maps/pr-101110-map.yaml:3]: Same question regarding the single quotes.

File: releases/release-1.22/release-notes/maps/pr-101129-map.yaml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+pr: 101129
+releasenote:
+  text: 'kubeadm: remove the deprecated kubeadm API `v1beta1`. Introduce a new kubeadm
```
Comment (erismaster) [inline releases/release-1.22/release-notes/maps/pr-101129-map.yaml:3]: Here as well.

File: releases/release-1.22/release-notes/maps/pr-101171-map.yaml [hunk 8]
```diff
@@ -0,0 +1,3 @@
+pr: 101171
+releasenote:
+  text: ' The new flag `--chunk-size=SIZE` has been added to `kubectl describe`. This flag may be used to alter the number of items or disable this feature when `0` is passed.'
```
Comment (erismaster) [inline releases/release-1.22/release-notes/maps/pr-101171-map.yaml:3]: And here, theres an extra space as well.

File: releases/release-1.22/release-notes/maps/pr-99336-map.yaml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+pr: 99336
+releasenote:
+  text: 'kubelet: improve the performance when waiting for a synchronization of the
```
Comment (erismaster) [inline releases/release-1.22/release-notes/maps/pr-99336-map.yaml:3]: Here too.

**Review by erismaster — CHANGES_REQUESTED** [review]

File: releases/release-1.22/release-notes/maps/pr-100694-map.yaml (hunk 5 above)
Comment (pmmalinov01) [reply releases/release-1.22/release-notes/maps/pr-100694-map.yaml:3]: Single quotes need to be used to prevent the extra `:` to be detected as YAML key.

**Pull request merged**